web/styles/popovers.css

```diff
@@ -81,32 +81,39 @@
     }
 }
 
-.tippy-box[data-theme="navbar-dropdown-menu"] {
-    border: 0;
+.tippy-box[data-theme="popover-menu"] {
+    background-color: var(--color-background-popover-menu);
+    border: 1px solid var(--color-border-popover-menu);
     border-radius: 6px;
+    box-shadow: var(--box-shadow-navbar-popover-menu);
 
     > .tippy-content {
         padding: 0;
     }
 
     > .tippy-arrow {
-        top: -5.5px;
-        width: 16px;
-        height: 8px;
-        z-index: 1;
-        filter: drop-shadow(0 -1.25px 0 var(--color-border-dropdown-menu));
+        color: var(--color-background-popover-menu);
+    }
 
-        &::before {
-            content: "";
-            top: 0;
-            width: 100%;
-            height: 100%;
-            mask-image: url("../shared/icons/popover-arrow.svg");
-            mask-size: contain;
-            mask-repeat: no-repeat;
-            color: var(--color-background-dropdown-menu);
-        }
+    &[data-placement^="top"] > .tippy-arrow::before {
+        bottom: -6.5px;
     }
+
+    &[data-placement^="bottom"] > .tippy-arrow::before {
+        top: -6.5px;
+    }
+
+    &[data-placement^="left"] > .tippy-arrow::before {
+        right: -6.5px;
+    }
+
+    &[data-placement^="right"] > .tippy-arrow::before {
+        left: -6.5px;
+    }
```

Comment:
We should also have comments explaining what these are doing, since it's overriding a bit of upstream CSS, I imagine.

Reply:
Ahh, OK, I see you add this in a later commit.

Reply:
Yes! Would you like me to merge the later commit with the former one?

Reply:
Nah, this is fine, it's just documentation.